webapp/client/public/css/variables.css

```diff
@@ -135,14 +136,19 @@
     --spacing-02: 0.625rem;
     --spacing-03: 1rem;
     --spacing-04: 1.25rem;
+    --spacing-04a: 1.375rem;
```

Comment:
I think i like the idea. Did you talk to design?

Reply:
I talked to design and we realized we have some spacing missing in our variables, therefore I added the ones with *a. We decided to move on like this for this ticket and we will then have a session with design where we update the variables so that we don't have this hacks anymore